### PR TITLE
First pull request for Portfolio website

### PR DESCRIPTION
**Pull request by nitinramadoss**
Add Profile photo and creates basic layout for the home page of the website

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,14 +2,32 @@
...
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Nitin's Portfolio</h1>
+      <marquee>Latest Update: Learning web app development and practicing machine learning</marquee>
+      <hr style="height:1px;border-top:1px solid #0000FF" />
+      <img src="images/profile.jpg" alt="Profile Photograph">
+      <h3>University of Florida Bachelors in Computer Science</h3>
+      <h3>Google STEP Intern</h3>
+      <p>I have been learning how to program for roughly 6 years now. I began with learning web development for an FBLA competition and I now build android applications for national competitions like PennApps and for my personal developer account on the Play Store. Software development is one of my favorite subfields in computer science. Throughout my junior and senior year in high school, I taught students how to program in a computer science club that I founder. My goal is to lead emerging technologies and build software that can make a postive impact in society.</p>
```
Comment (lee12jin) [inline portfolio/src/main/webapp/index.html:25]: For long paragraphs like this, you should multiline them.
\<p>
[indent]I have been learning...
\</p>

Your text editor might have maximum spacing settings, generally stops at 80 characters. Going beyond that can get hard to read.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,14 +2,32 @@
...
+      <hr style="border-top: dotted 1px;" />
+      <h4>Extra Links</h4>
+      <a href="https://www.linkedin.com/in/nitin-r-b8aaa4107/">Connect with me on Linkedin!</a>
+      <p>  </p>
```
Comment (lee12jin) [inline portfolio/src/main/webapp/index.html:29]: Did you want an empty paragraph here?
If you wanted a new line, you can use <br>

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (nitinramadoss) [reply portfolio/src/main/webapp/index.html:25]: I will do that in the next pull request, thank you!

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (nitinramadoss) [reply portfolio/src/main/webapp/index.html:29]: Yeah I think <br> was the tag I was looking for, thanks!

**Pull request merged**